event_stage_cancelled/tests/test_event_stage_cancelled.py

```diff
@@ -65,7 +65,7 @@ def test_event_cancellation(self):
         self.assertFalse(bool(self.event_mail.mail_registration_ids))
         self.assertFalse(self.event_mail.scheduled_date)
         self.assertEqual(self.event_mail.mail_state, "running")
-        # Inject bypass_reason for test compatibility with event_registration_cancel_reason
+        # Inject bypass_reason for test compatibility with event_registration_cancel_reason  # noqa: E501
```

Comment:
In this case I think it is not necessary to ignore the linter as it is a simple comment and does not influence the logic. Split the comment and that's it